docs/development/environment.md

```diff
@@ -0,0 +1,22 @@
+# Environment
+
+## Development
+
+This plugin contains both a server and web app portion.
+
+Use `make dist` to build distributions of the plugin that you can upload to a Mattermost server. 
+
+Use `make check-style` to check the style. 
+
+Use `make deploy` to deploy the plugin to your local server.  Before running `make deploy` you need to set a few environment variables:
```

Comment:
```suggestion
Use `make deploy` to deploy the plugin to your local server. Before running `make deploy` you need to set a few environment variables:
```

Reply:
A long long time ago, somewhere I had learned that there should be two spaces after a period to emphasize the end of a sentence.  Since then I got into the automatic habit of doing that.  Of course, no one really does that, even if they did so at one time.  I guess I will have to watch myself and stop putting that extra space :-)